docs/developer-guide/api-user-guide.md

```diff
@@ -0,0 +1,286 @@
+| /v2/{repo}/referrers/{digest}| GET | Lists referrers given a digest | 
+| /v2/{repo}/tags/list | GET | List all image tags in a repository | 
+
+
+### zot OCI extension endpoints
+
+| Endpoint | Actions | Description | Availability |
+| -------- | ------- | ----------- | -------------|
+| /v2/_zot/ext/mgmt| GET | Mgmt extension endpoints | Enabled by using the `mgmt` build label and enabling the `search` extension in the configuration file. |
+| /v2/_zot/ext/notation| POST | With query parameters, uploads certificates for signature verification | Enabled by using the `imagetrust` build label and enabling the `trust` extension with the `notation` option enabled. |
+| /v2/_zot/ext/search |  | Enhanced search | Enabled by using the `search` build label and enabling the `search` extension in the configuration file. |
+| /v2/_zot/ext/cosign| POST | Uploads keys for signature verification | Enabled by using the `imagetrust` build label and enabling the `trust` extension with the `cosign` option enabled. |
+| /v2/_zot/ext/userprefs| PUT | User preferences endpoints | Enabled by using the `userprefs` build label and enabling both the `search` and the `ui` extensions in the configuration file. |
+
+
+### zot auth endpoints
+
+| Endpoint | Actions | Description | Availability |
+| -------- | ------- | ----------- | -------------|
+| /auth/apikey| DELETE, GET, POST | Creates, lists, or deletes API keys | Available when API key authentication is enabled in the configuration file (`"apikey": true`). |
+| /auth/login | POST  | Opens an API session | Available when authentication is available. This includes not only OpenID, but all session-based authentication. |
+| /auth/logout| POST  | Ends an API session | Available when authentication is available. This includes not only OpenID, but all session-based authentication. 
+| /auth/callback/\<provider\> | POST | Specifies a social authentication service provider for redirecting logins, such as Google or dex. | Enabled when an OpenID authentication service provider is specified in the configuration file. |
+
+
+### other zot endpoints
+
+| Endpoint | Actions | Description | Availability |
+| -------- | ------- | ----------- | -------------|
+| /v2/_zot/pprof/ | GET | Returns a current HTML-format profile list along with a count of currently available records for each profile. See [Performance Profiling in zot](../articles/pprof.md) for usage details. | Always enabled. |
+| /v2/_zot/debug/graphql-playground# |  | See [Using GraphQL](../articles/graphql.md) for details. | Enabled only in a `binary-debug` zot build or when the zot registry has been built with the `debug` extension label. |
+
+
+### ORAS endpoints
+
+| Endpoint | Actions | Description | Availability |
+| -------- | ------- | ----------- | -------------|
+| /oras/artifacts/v1/_zot/ \ manifests/{digest}/referrers | GET | [OCI Registry As Storage (ORAS)](https://oras.land/) endpoints | Always enabled. |
```

Comment:
This is supposed to be `/oras/artifacts/v1/{repo}/manifests/{digest}/referrers`

Reply:
Fixed